recipes/msys2/all/conandata.yml

```diff
@@ -1,34 +1,7 @@
 sources:
-  "20190524":
-    "x86_64":
-          url: [
-                 "http://repo.msys2.org/distrib/x86_64/msys2-base-x86_64-20190524.tar.xz",
-                 "https://sourceforge.net/projects/msys2/files/Base/x86_64/msys2-base-x86_64-20190524.tar.xz",
-          ]
-          sha256: "168e156fa9f00d90a8445676c023c63be6e82f71487f4e2688ab5cb13b345383"
-    "x86":
-          url: [
-                 "http://repo.msys2.org/distrib/i686/msys2-base-i686-20190524.tar.xz",
-                 "https://sourceforge.net/projects/msys2/files/Base/i686/msys2-base-i686-20190524.tar.xz",
-          ]
-          sha256: "7a463afae8bf6ce8262f010a9a1648d056ad5cefc66d6eb69fe948c57d4ccb53"
-  "20200517":
-    "x86_64":
-          url: [
-                 "http://repo.msys2.org/distrib/x86_64/msys2-base-x86_64-20200517.tar.xz",
-                 "https://sourceforge.net/projects/msys2/files/Base/x86_64/msys2-base-x86_64-20200517.tar.xz",
-          ]
-          sha256: "c4443113497acb2d2e285d40b929fc55f33f8f669902595ecdf66a655b63dc60"
-    "x86":
-          url: [
-                 "http://repo.msys2.org/distrib/i686/msys2-base-i686-20200517.tar.xz",
-                 "https://sourceforge.net/projects/msys2/files/Base/i686/msys2-base-i686-20200517.tar.xz",
-          ]
-          sha256: "4dbdde708a4bcf3c056e8f4f28b1b0e7a3210ffacc6296a7d11fdaf076fd431e"
-  "20210105":
-    "x86_64":
-          url: [
-                 "http://repo.msys2.org/distrib/x86_64/msys2-base-x86_64-20210105.tar.xz",
-                 "https://sourceforge.net/projects/msys2/files/Base/x86_64/msys2-base-x86_64-20210105.tar.xz",
-          ]
-          sha256: "982e54de087d53adfc6a8caf7614d4a7add36dd02dcb0b7838060dd893e9f596"
+  "cci.latest":
```

Comment:
what is the consequences for CCI recipes depending on removed versions?

Reply:
Existing binaries will remain available, and their hashes should not change because this recipe does not have requirements.
see also https://github.com/conan-io/conan-center-index/pull/4286#issuecomment-796737851